conda-reqs/chipyard.yaml

```diff
@@ -1,5 +1,6 @@
 channels:
     - ucb-bar
+    - litex-hub
```

Comment:
What packages are using this channel? (If none are using this, we should delete it)

Reply:
Actually, nevermind I see what was done. You added this to all files so that `conda-lock` wouldn't error.